### DISCUSSION TITLE
Understanding query performance reports in Supabase

### DISCUSSION THREAD

**Discussion by MrVibe** [supabase/supabase#30436] [Questions] [answered]
I apologise firstly if this is a noob question, I am still learning supabase.
I wanted to check the slowest queries in my supabase and found a query performance section.

Unfortunately, I can not locate the queries displayed in my codebase :
https://prnt.sc/EISNQMTmcD4e

I suspect these are queries used internally by supabase, how can we improve these queries ? Do we need to be concerned about it ?

**Comment by GaryAustin1** [comment, marked as answer]
There is not much you can do for default queries needed by the REST API (authenticator), realtime, pg_net, cron etc.  

That is a PosgREST (REST API) query of a standard Postgres view https://www.postgresql.org/docs/15/view-pg-timezone-names.html and I see comments on the web it is slow.  I looked at my instance though and it is called rarely.
